app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -58,9 +64,9 @@
             android:layout_marginTop="4dp"
             android:layout_marginEnd="8dp"
             android:ellipsize="end"
-            android:fontFamily="sans-serif-medium"
+            android:fontFamily="sans-serif"
             android:maxLines="2"
-            android:text="@{String.format(@string/chapter_name, (viewModel.index + 1), viewModel.name)}"
+            android:text="@{titleContent}"
```

Comment:
Why are you updating the title here. The issue is only related to the borders right.

Reply:
yeah, but also if some pre-requisites are not completed, the title has to be changed to - "to unlock this chapter, you have to finish this chapter", hence I used the title here